reactjs-films-homework/src/components/Header/__tests__/Header.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { cleanup, render, screen } from '@testing-library/react';
+import Header from '../Header';
+
+describe('Header', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('Header snapshot', () => {
```

Comment:
ditto

Reply:
done, test refactored, and snapshot fixed